pkg/clients/tekton/pipelineruns.go

```diff
@@ -239,7 +239,7 @@ func (t *TektonController) StorePipelineRun(prefix string, pipelineRun *pipeline
 
 	pipelineRunYaml, err := yaml.Marshal(pipelineRun)
 	if err != nil {
-		return err
+		g.GinkgoWriter.Printf("failed to store pipelineRun %s:%s: %s\n", pipelineRun.GetNamespace(), pipelineRun.GetName(), err.Error())
```

Comment:
Shouldn't you still return the error after this in case tests are checking if this function returns error?

Reply:
To help debug, ignore the errors to store as many logs as it could.

Reply:
@jinqi7 would it be better to print out only [this error](https://github.com/konflux-ci/e2e-tests/pull/1474/files#diff-882ad83f7873be0bfce19a5ed625d60a4430f334e4d3eff4022a0662510d8e1cR235-R237) instead?

Reply:
I want to make the change in Line#236.

Reply:
The error message may be misleading. Changed it.